### PR TITLE
Node stress test client and metrics client

### LINKED ISSUE #5988: Create a stress test client for Node.js
The specification for the stress test client is at : https://github.com/grpc/grpc/blob/master/tools/run_tests/stress_test/STRESS_CLIENT_SPEC.md

FYI: The C++ stress test client is [here](https://github.com/grpc/grpc/blob/master/test/cpp/interop/stress_test.cc)

### PR DESCRIPTION
**Pull request by murgatroid99**
This fixes #5988.

### CODE REVIEW

**Comment by sreecha** [comment]
The branch seems out of date and needs a merge with latest master

File: src/node/stress/stress_client.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  var diff = process.hrtime(start_time);
+  var seconds = diff[0] + diff[1] / 1e9;
+  return {long_value: query_count / seconds};
+}
+
+function start(server_addresses, test_cases, channels_per_server,
+               parallel_calls_per_channel, metrics_port) {
+  running = true;
+  // Assuming that we are not calling unimplemented_method
+  var Client = interop_client.test_cases.empty_unary.Client;
```
Comment (sreecha) [inline src/node/stress/stress_client.js]: Just curious: What is the purpose of this line ?

Comment (murgatroid99) [inline src/node/stress/stress_client.js]: It just has to do with the way I organized my interop client. Technically, it has support for different client types for different tests, and I want to indicate that here I'm explicitly using only the client type that corresponds to `empty_unary` (which is also currently the client type used by everything but `unimplemented_method`).

Comment (sreecha) [inline src/node/stress/stress_client.js]: Ah. Understood. Can you add a small comment that clarifies that this Client type isn't just for empty_unary and can be used for all other test cases as well.

Comment (murgatroid99) [inline src/node/stress/stress_client.js]: I have done that.

Comment (sreecha) [inline src/node/stress/stress_client.js]: Great!. Thanks

**Comment by sreecha** [comment]
LGTM. Will add the `ready to merge` label once the tests pass

**Comment by murgatroid99** [comment]
The only test failure is a C# coverage timeout. I don't think that's related.

**Pull request merged**